src/controllers/subscription.js

```diff
@@ -1,10 +1,24 @@
 import Subscription from '../models/Subscription';
+import Class from '../models/Class';
+import Member from '../models/Member';
+import member from '../seeds/member';
 
 const updateSubscription = async (req, res) => {
   try {
     const { id } = req.params;
     const { isActive } = await Subscription.findById(id);
 
+    if (member) {
```

Comment:
isnt this the same as the code in line 40?

Reply:
oh yes, I delete it there